locale/en/download/package-manager.md

```diff
@@ -211,6 +212,14 @@ from source:
 nvm uninstall 8
 ```
 
+## fnm
+
+Fast and simple Node.js version manager built-in Rust used to manage multiple released Node.js versions. It allows you to perform operations like install, uninstall, switch Node versions automatically based on the current directory, etc.
+To install fnm, use this [install script](https://github.com/Schniz/fnm#using-a-script-macoslinux).
+
+fnm has cross-platform support (macOS, Windows, Linux) & all poplar shells (Bash, Zsh, Fish, PowerShell, Windows Command Line Prompt)
+it's built with speed in mind and compatibility support for `.node-version` and `.nvmrc` files.
+
```

Comment:
Similarly, the sections are in case-insensitive alphabetical order, so this should be added earlier in the document.

Reply:
I’m on it 🙌

Reply:
Done ✅